src/atoms/Space/Space.native.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
+
+import { getPxScale } from '../../_helpers/theme';
+
+const Space = styled(({ margin, padding, children, ...props }) =>
+  React.cloneElement(children, props),
```

Comment:
Where is it throwing an error where
```
<Space>
  <Text>Some text</Text>
  <Text>Some Other Text</Text>
</Space>

```

Ideally the expected scenario is this
```
<Space>
	<View>
	  <Text>Some text</Text>
	  <Text>Some Other Text</Text>
	</View>
</Space>

```

Reply:
Created Space as functional component and throws error if !(children.length ==1)